submariner/cable-driver-protocols-enhancement.md

```diff
@@ -0,0 +1,155 @@
+# Cable Driver Protocol Support Enhancements
+
+## Summary
+
+Submariner today supports three cable drivers: VXLAN, IPSec (tunnel mode) and WireGuard. The
+goal of this proposal is to outline additional protocols and enhancements for the Submariner
+Cable Driver. These include the additional support of IP-in-IP for unencrypted connections
+between clusters and the support of IPSec transport mode.
+
+## Proposal
+
+### IP-in-IP Encapsulation
+
+This enhancement proposes to support IP-in-IP for un-encrypted connection between clusters. A
+new IP tunnelling interface `ipip-tunnel` will be added to the active gateway node of each cluster
+and it will act as the Tunnel End Point. The traffic which is destined to other clusters will be
+forwarded via this interface to the appropriate clusters.
+
+> **_NOTE:_** A single IP Tunnelling interface will be added, and tunnels managed using
+`# ip route add encap ip <args>` in order to avoid a point to point connection per tunnel.
+
+#### Design Details
+
+* A new interface "ipip-tunnel" will be added.
+* The IP tunnelling interface will be one-to-many, that is a single interface will be used to
+connect to all the joined clusters.
+* For the "ipip-tunnel" IP, the prefix "242" will be used, followed by the rest of the blocks
```

Comment:
Lets avoid 242 prefix as its the default [Globalnet CIDR](https://github.com/submariner-io/submariner-operator/blob/16df4bcd7bc55cb87d6d80adbb7d5a726749a715/pkg/broker/globalcidr_cm.go#L39) and the same ip could be allocated by Globalnet controller as Egress/IngressIP.

VxLAN currently uses 241 prefix, so we can use 243.